lisa/operating_system.py

```diff
@@ -636,6 +640,12 @@ def name_pattern(cls) -> Pattern[str]:
         return re.compile("^CentOS|Centos|centos|clear-linux-os$")
 
 
+class AlmaLinux(Redhat):
```

Comment:
We don't need a class for AlmaLinux so far, because it has no special logic than redHat